database/migrations/2020_11_12_174543_new_bristol_stand.php

```diff
@@ -21,8 +21,8 @@ public function up()
                 [
                     'airfield_id' => DB::table('airfield')->where('code', 'EGGD')->first()->id,
                     'identifier' => '7N',
-                    'latitude' => 'N051.23.10.242',
-                    'longitude' => 'W002.42.30.128',
+                    'latitude' => 51.38617833,
+                    'longitude' => -2.70836889,
```

Comment:
Why are we modifiying a migration that has already ran in production?

Reply:
Due to the timestamps on the migration, it occurs after `2020_10_31_171409_add_decimal_lat_long_to_stands.php` - which converts all the stand coordinates to decimal degrees from UKSF coordinates (better for performance if pre-converted). It won't affect prod and the stand added in this migration will get converted like all the others, but if we don't change this migration, it'll freak out in dev when you rebuild fresh.